client.coffee

```diff
@@ -66,7 +66,7 @@ renderButton = (buffer) !->
 
 renderFunny = !->
 	Obs.observe !->
-		funny = Db.personal.get('funnies')
+		funny = Db.personal.get('funnies', 'current')
```

Comment:
Ik heb even alle funnies info in een objectje gezet, dus "laatste click waarbij hij een funny zag", "laatste tijd daarvan", "huidige funny" (if any), etc.. Dan staat het mooi bij mekaar :)